apple/internal/ios_rules.bzl

```diff
@@ -290,6 +290,7 @@ def _ios_application_impl(ctx):
         rule_descriptor = rule_descriptor,
         rule_executables = rule_executables,
         rule_label = label,
+        codesignopts = [ctx.expand_make_variables("codesignopts", opt, {}) for opt in ctx.attr.codesignopts],
```

Comment:
the variable expansion here is pretty annoying, im kinda surprised we're not doing this anywhere else? maybe it isn't necessary?

Reply:
This was a workaround because there's nothing like `codesign_inputs`, so `expand_locations` doesn't work. This works because codesigning is always done outside of a sandbox.

Reply:
nice. i guess we'd prob need to abstract this if we did this on every possible rule

Reply:
Note for remote exec to be compatible with this we might eventually need codesign_inputs as well, a problem for another day